### ISSUE TITLE
Discriminator output oscillating between 0 and 1

### ISSUE THREAD

**Issue by shamitlal** [yenchenlin/pix2pix-tensorflow#28]
Hi
I trained your code on the facades dataset. However the discriminator outputs for real and fake images i.e. self.D and selfD_ keeps on oscillating from 0 to 1 to 0. In ideal case, they should both converge to 0.5. However even after sufficient number of training epochs, these two outputs are not converging at all and keeps on oscillating.
Does this mean that discriminator is not getting trained at all?
If discriminator is not getting trained well, is the final image to image translation only due to L1 loss optimization?

Thanks
<img width="270" alt="screen shot 2017-03-21 at 3 13 02 am" src="https://cloud.githubusercontent.com/assets/9249746/24123149/64403d52-0de4-11e7-9817-b7711ec023d2.png">
